### PR TITLE
Add cmd e2e:xvfb to execute e2e tests in container

### PR DESCRIPTION
**Pull request by dsimansk**
Related to #93. This PR adds possibility to execute e2e tests from Docker container.

### CODE REVIEW

**Comment by dsimansk** [comment]
To function properly this PR depends on #368.

File: e2e-xvfb.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+nohup /usr/bin/Xvfb :99 -ac -screen 0 1280x1024x24 &
+export DISPLAY=:99
+yarn
+npm rebuild node-sass
```
Comment (jludvice) [inline e2e-xvfb.sh]: Do we actually need `npm rebuild node-sass` step?
We don't use local ipaas-ui right?

File: e2e/conf/protractor.base.conf.js [hunk 2]
```diff
@@ -18,7 +18,10 @@ exports.config = {
     testPath +  '/**/*.feature'
   ],
   capabilities: {
-    'browserName': 'chrome'
+    'browserName': 'chrome',
+    'chromeOptions': {
+      'args': ['--no-sandbox']
+    }
```
Comment (jludvice) [inline e2e/conf/protractor.base.conf.js]: is this option common to all protractor configurations ?
If it's just for `yarn e2e:ipaas-qe`, we might move `chromeOptions` to `e2e/conf/protractor.ipaas-qe.conf.js`

File: e2e-xvfb.sh (hunk 1 above)
Comment (dsimansk) [inline e2e-xvfb.sh]: Well, not sure about that. We might actually need `yarn webdriver-manager update`.

File: e2e/conf/protractor.base.conf.js (hunk 2 above)
Comment (dsimansk) [inline e2e/conf/protractor.base.conf.js]: I'd keep it common for know, if there aren't any objections :)

File: e2e-xvfb.sh [hunk 3]
```diff
@@ -3,5 +3,5 @@
 nohup /usr/bin/Xvfb :99 -ac -screen 0 1280x1024x24 &
 export DISPLAY=:99
 yarn
-npm rebuild node-sass
+yarn webdriver-manager update
```
Comment (jludvice) [inline e2e-xvfb.sh]: good catch !

**Comment by jludvice** [comment]
Update: #368 is merged

File: e2e-xvfb.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+nohup /usr/bin/Xvfb :99 -ac -screen 0 1280x1024x24 &
```
Comment (kahboom) [inline e2e-xvfb.sh]: Just for reference, the projector for RH Summit has a 16:9 aspect ratio and 1280×720 resolution.

File: e2e/conf/protractor.base.conf.js (hunk 2 above)
Comment (kahboom) [inline e2e/conf/protractor.base.conf.js]: I agree with @jludvice , probably best to move to `e2e/conf/protractor.ipaas-qe.conf.js` eventually, but for now is not a big deal at all. 👍

**Pull request merged**